### PR TITLE
Mirror cleanup and enhancements

### PR DESCRIPTION
**Pull request by bfournie**
Various cleanup and changes based on review and discussion

### CODE REVIEW

**Comment by andfasano** [comment]
Thanks @bfournie for these changes, as they improve considerably the OC_MIRROR feature. I've still pending a couple of non blocking points:
* Automatically changing `MIRROR_IMAGES` when `OC_MIRROR` is set still sounds a little bit weird to me, given that the OC_MIRROR indicates _a way_ to perform the mirroring (in contrast to `oc adm mirror ...` approach)
* When running locally, the `.oc-mirror.log` file is still getting produced in the _current directory_, and not in the working dir

/approve

**Comment by bfournie** [comment]
> Thanks @bfournie for these changes, as they improve considerably the OC_MIRROR feature. I've still pending a couple of non blocking points:
>

> * When running locally, the `.oc-mirror.log` file is still getting produced in the _current directory_, and not in the working dir
> 
> /approve



> Thanks @bfournie for these changes, as they improve considerably the OC_MIRROR feature. I've still pending a couple of non blocking points:
> 
> * Automatically changing `MIRROR_IMAGES` when `OC_MIRROR` is set still sounds a little bit weird to me, given that the OC_MIRROR indicates _a way_ to perform the mirroring (in contrast to `oc adm mirror ...` approach)

Changed OC_MIRROR to MIRROR_COMMAND to identify the command used to do the mirroring. MIRROR_IMAGES will be needed to be set

> * When running locally, the `.oc-mirror.log` file is still getting produced in the _current directory_, and not in the working dir
> 

Fixed one spot where oc-mirror was not being run in WORKING_DIR

> /approve

**Comment by bfournie** [comment]
Please review, I don't plan on any more changes to this PR but will have a follow-on PR to remove the `OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE` setting in general

File: config_example.sh [hunk 1]
```diff
@@ -559,17 +559,18 @@ set -x
...
+# Default: oc-adm
+#export MIRROR_COMMAND=oc-adm
+
+# When the MIRROR_COMMAND is set to 'oc-mirror' the auths for the mirror will be added
+# to DOCKER_CONFIG_FILE or an UNAUTHORIZED error will result.
+# An example entry in this file is:
+# "virthost.ostest.test.metalkube.org:LOCAL_REGISTRY_PORT=": { "auth": "<auth string>" },
 #export DOCKER_CONFIG_FILE=$HOME/.docker/config.json
 
 # MIRROR_OLM -
```
Comment (lranjbar) [inline config_example.sh:576]: MIRROR_OLM isn't supported by the `oc mirror` command yet.

**Review by lranjbar — COMMENTED** [review]
We should just call out the gap with `oc mirror` command and MIRROR_OLM. In the future we should refactor our MCE install to use MIRROR_OLM. But this is a follow up story and isn't high priority.

**Pull request merged**